sdks/RepoToolset/tools/Directory.Build.props

```diff
@@ -0,0 +1,4 @@
+<!-- Copyright (c)  Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information. -->
+<Project>
+  
```

Comment:
Add a comment why this file is needed

Reply:
Is the intent to stop the parent dir.build.props file from loading or to completely isolate this build? If it's the latter more is needed to cut off user customizations of build.

Reply:
The main intent is to stop the parent Directory.build files from being used, so that in another project using repo toolset you can do something like this:

```xml
<RepoToolsetDir>C:\git\roslyn-tools\sdks\RepoToolset\tools\</RepoToolsetDir>
```

And then you can test out changes to repo toolset locally.